src/features/Students/data/api.js

```diff
@@ -0,0 +1,19 @@
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform';
+
+function getStudentbyInstitutionAdmin(page) {
+  const params = { page };
+
+  Object.keys(params).forEach((key) => {
+    if (params[key] === '') {
+      delete params[key];
+    }
```

Comment:
```suggestion
    };
```

Reply:
@anfbermudezme unncessery semicolon according to Lint.